### ISSUE TITLE
Different logging levels for appenders in same logger

### ISSUE THREAD

**Issue by ShehabElsayed** [johnmyleswhite/log4r#20]
Hi,

Is there a way to have 2 appenders in the same logger but with different logging levels? For example, can I add a console appender with level INFO and a file appender with level DEBUG to the same logger such that the results would be like the following?

* `info(logger, "Hello")`: would write Hello to both console and file
* `debug(logger, "Hello")`: would only write Hello to file

Thank you very much in advance.

**Comment by atheriel** [comment]
I don't think this is possible with the current API, as appenders do not have separate thresholds.

But it's almost certainly possible to accomplish this:

1. Write a custom console appender that handles the level internally and drops `DEBUG` messages.
2. Set the logger's overall threshold at `DEBUG` use the existing file appender.
